go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/docker/docker v23.0.8+incompatible
 	github.com/fatih/color v1.15.0
 	github.com/go-git/go-billy/v5 v5.5.0
+	github.com/go-git/go-git v4.7.0+incompatible
```

Comment:
Is this intentional? It seems to be bringing in an older version of go-git alongside v5.